ui-core/src/components/inputs/StatusMessage.tsx

```diff
@@ -18,11 +20,19 @@ export type StatusMessageProps = {
 };
 
 const StatusMessage = ({ statusWithMessage, showIcon, small }: StatusMessageProps) => {
-  const { status, message } = statusWithMessage;
+  const { tooltip, status, message } = statusWithMessage;
   if (message === undefined) return null;
 
   return (
-    <div className="status-message-wrapper">
+    <div
+      className={cx({
+        'status-message-wrapper': !tooltip,
+        'status-message-wrapper-tooltip': tooltip,
+        'tooltip-left': tooltip === 'left',
+        'tooltip-right': tooltip === 'right',
+        [status]: status,
```

Comment:
This doesn't seem to be enough, the text and background color change only for the error status